CONTRIBUTING.md

```diff
@@ -10,18 +10,33 @@ with `git commit -s`.
 
 ## Contribution Flow
 
+### Forking
```

Comment:
This is kind of redundant, since forking is the standard way to contribute for external contributors, but I'm ok with being verbose.